webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    main: path.resolve(__dirname, 'src/index.js'),
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      title: 'To-Do List',
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].[contenthash].js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+    // publicPath: '/',
```

Comment:
I kindly suggest you remove all commented code as this does not follow the [YANGI] 🧹  (https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md) and adds space to you bundles 👍

Reply:
[Removed comments](https://github.com/Lucash2022/To-Do-list-review/pull/1/commits/666b72aa7e86ee3da8e1d9279afd949f1186ab7c)